products/mcp/typescript/src/tools/errorTracking/errorDetails.ts

```diff
@@ -12,6 +12,7 @@ export const errorDetailsHandler = async (context: Context, params: Params) => {
 
     const errorQuery = {
         kind: 'ErrorTrackingQuery',
+        orderBy: 'occurrences',
```

Comment:
```suggestion
        orderBy: 'occurrences',
        withAggregations: false,
```
That will speed up a bit the query